test/bigtest/tests/line-edit-test.js

```diff
@@ -27,8 +27,8 @@ import OrderDetailsPage from '../interactors/order-details-page';
 import { CalloutInteractor } from '../interactors';
 
 const TITLE = 'TEST_VALUE';
-const requiredField = 'Required!';
-const validationYearMessage = 'Field should be 4-digit year';
+// const requiredField = 'Required!';
```

Comment:
should they be removed?

Reply:
Tests with these consts were commented during migration to react-final-form. So we need to come back to them later

Reply:
@AndreiShumski2 you're right, butt what the reason of deletion of this commented out code now?

Reply:
It looks weird since you've deleted part of commented out code, and what is the reason to touch this at all?

Reply:
I've just commented unused vars to avoid eslint warnings

`30:7  warning  'requiredField' is assigned a value but never used          no-unused-vars`
`31:7  warning  'validationYearMessage' is assigned a value but never used  no-unused-vars`

Reply:
yes, you're right, commenting is fine